examples/contracts/src/lib.rs

```diff
@@ -668,7 +668,11 @@ mod tests {
         let _ = contract_instance
             .methods()
             .initialize_counter(42)
-            .add_custom_asset(BASE_ASSET_ID, amount, Some(other_wallet.address().clone()))
+            .add_custom_asset(
+                AssetId::default(),
```

Comment:
Nit: Since it's in the docs. It's a bit more descriptive and it might be useful to familiarize users with it.

```suggestion
                AssetId::BASE,
```

Reply:
I wanted to avoid using the name `BASE` because it could be misleading. I tried to stick to `default` in our tests as the focus is not on the asset ID.